strawberry_django_plus/permissions.py

```diff
@@ -49,7 +49,7 @@
     from .integrations.guardian import get_user_or_anonymous
 
     get_user_or_anonymous = resolvers.async_safe(get_user_or_anonymous)
-except ImportError:
+except (ImportError, RuntimeError):
     # Access the user's id to force it to be loaded from the database
     get_user_or_anonymous = resolvers.async_safe(lambda u: (u, u.id)[0])
 
```

Comment:
I can't comment on lines you haven't change, but check that pyright is saying that a lot of `casts` are not required anymore. You can probably remove those